### ISSUE TITLE
Allow user to choose which text input to enter upon 'gi'

### ISSUE THREAD

**Issue by otterlover24** [tridactyl/tridactyl#3209]
Hi,
I am a very happy user of Tridactyl. It has almost allowed me to completely use my keyboard instead of the mouse for navigating websites. However, one issue is that when I want to go to a text input field, 'gi' simply goes to the one it chooses, rather than allowing me to choose which text input field to go to. Ideally, there should be a popup with letters for each text field upon pressing 'gi', and pressing the appropriate key brings me into that text field.

Tridactyl version: 1.20.4
Firefox version: Mozilla Firefox 85.0
Operating system: linux

*   Brief description of the problem:

*   Steps to reproduce:

    1.  2.  3.  4.  5.

*   URL of the website the bug happens on:

*   Config (in a new tab, run `:viewconfig`, copy the url and paste it somewhere like gist.github.com):

*   Contents of ~/.tridactylrc or ~/.config/tridactyl/tridactylrc (if they exist):

```
Insert tridactylrc contents between the backticks
```

*   Result of running `:! echo $PATH`, or `! echo %PATH%` on Windows, in Tridactyl:
*   Unix-like only: result of running `printf '%c\0\0\0{"cmd": "run", "command": "echo $PATH"}' 39 | ~/.local/share/tridactyl/native_main.py` in a terminal:

**Comment by bovine3dom** [comment]
There's two solutions in Tridactyl currently

- press `<Tab>` after pressing `gi` to cycle inputs, or
- use `:js tri.excmds.hint("-c",tri.excmds.INPUTTAGS_selectors)` to pick a visible input. If you decide you like this a lot you can replace `gi` with `bind gi js tri.excmds.hint("-c",tri.excmds.INPUTTAGS_selectors)`

We could add a default bind for the second option like we have for the other hint modes, but we're running out of letters we can use after `;` (just look at `:bind ;` and press and hold `<Tab>` to se what I mean...)

**Comment by Rummskartoffel** [comment]
`gI` would be available and wouldn't be *too* unintuitive, maybe we could use that?

**Comment by bovine3dom** [comment]
yeah, OK. Bonus points if you can put the user in `input` mode afterwards _and_ handle cancelling picking a hint by pressing Escape without everything breaking : )

**Comment by Rummskartoffel** [comment]
I *might* take a shot at it, but I'm not promising anything.

**Comment by protrolium** [comment]
could also invoke hint mode `f` and select appropriate field ?

**Comment by bovine3dom** [comment]
@protrolium that's what `js tri.excmds.hint("-c",tri.excmds.INPUTTAGS_selectors)` does essentially
